docs/README.md

```diff
@@ -9,7 +9,8 @@ Orbs are packages of reusable CircleCI configuration. They allow you to create e
 1. [Getting started with new config features](getting-started.md) 
 1. [Orb structure](structure.md)
 1. [Using orbs](using-orbs.md)
-1. [Authoring Orbs](orbs-authoring.md)
+1. [Testing orbs](orbs-testing.md)
+1. [Authoring and Publishing Orbs](orbs-authoring.md)
```

Comment:
Please swap the ordering, so authoring is before testing (and, in case they don't already, let's be sure those link to each other)